redbot/core/cli.py

```diff
@@ -74,6 +74,18 @@ def confirm(text: str, default: Optional[bool] = None) -> bool:
     return token
 
 
+def positive_int(arg: str) -> int:
+    try:
+        x = int(arg)
+    except ValueError:
+        raise argparse.ArgumentTypeError("Message cache size has to be a number.")
+    if x < 1000:
+        raise argparse.ArgumentTypeError("Message cache size has to be greater than 1000.")
```

Comment:
Either change "has to be greater than" to "can't be lower than" or to "has to be greater than or equal to"
```suggestion
        raise argparse.ArgumentTypeError("Message cache size can't be lower than 1000.")
```